src/MongoDB.Driver.Core/MongoDB.Driver.Core.targets

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Strictly speaking, we may not need this file if we keep the `content` and `contentFiles` in the nuget package, based off what Dima and I saw when testing, but we haven't invesetigated this thoroughly and we know this way works (C# FLE bindings approach).